tests/src/test/scala/org/http4s/headers/SetCookieHeaderSpec.scala

```diff
@@ -5,22 +5,23 @@
  */
 
 package org.http4s
-package parser
+package headers
 
+import cats.syntax.all._
 import org.http4s.headers.`Set-Cookie`
 import org.specs2.mutable.Specification
 
-class SetCookieHeaderSpec extends Specification with HeaderParserHelper[`Set-Cookie`] {
-  def hparse(value: String): ParseResult[`Set-Cookie`] = HttpHeaderParser.SET_COOKIE(value)
+class SetCookieHeaderSpec extends Specification {
+  def parse(value: String): `Set-Cookie` = `Set-Cookie`.parse(value).valueOr(throw _)
 
   "Set-Cookie parser" should {
     "parse a set cookie" in {
       val cookiestr =
-        "myname=\"foo\"; Domain=example.com; Max-Age=1; Path=value; SameSite=Strict; Secure;HttpOnly"
+        "myname=\"foo\"; Domain=example.com; Max-Age=1; Path=value; SameSite=Strict; Secure; HttpOnly"
```

Comment:
Like `Cookie`, we got stricter about whitespace around semi-colons.  Maybe relax this?